src/components/text/Link.jsx

```diff
@@ -1,61 +1,59 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+// @flow
+
+import * as React from 'react';
 import classNames from 'classnames';
+import Text from './Text';
+import {TEXT_TYPE, TEXT_COLOR, TEXT_SIZE, TEXT_WEIGHT, TEXT_TRANSFORM, TEXT_ALIGN} from './textConsts';
+import type {TextPropsType} from './Text';
 
-export const COLOR = {
-  GRAY: 'gray',
-  MINT: 'mint',
-  PEACH: 'peach',
-  LIGHT: 'light',
-  MUSTARD: 'mustard',
-  FINE_PRINT: 'for-fine-print',
-  FINE_PRINT_LIGHT: 'for-fine-print-light'
-};
+export const LINK_COLOR = TEXT_COLOR;
+export const LINK_TYPE = TEXT_TYPE;
+export const LINK_SIZE = TEXT_SIZE;
+export const LINK_WEIGHT = TEXT_WEIGHT;
+export const LINK_TRANSFORM = TEXT_TRANSFORM;
+export const LINK_ALIGN = TEXT_ALIGN;
 
-export const SIZE = {
```

Comment:
This is not backward-compatible

Reply:
It's not, but in many places we anyway need to make import like:
`import Link, {SIZE as LINK_SIZE} from 'style-guide/text/Link`
You think is does make seanse to fix it now?